<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationToolClass</class>
 <widget class="QMainWindow" name="CalibrationToolClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibrationTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="imageList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>250</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openCam">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takePic">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>take</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCam">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageWindow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#imageWindow {
    border: 1px solid black;
    padding: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CalibrationTool.qrc"/>
 </resources>
 <connections/>
</ui>
